### PR TITLE
drivers: wifi: Update peer TWT capability status

### PR DESCRIPTION
**Pull request by ajayparida**
[SHEL-1139] Update peer TWT capability info from connection info.

### CODE REVIEW

File: drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_mgmt.c [hunk 1]
```diff
@@ -584,6 +584,8 @@ int wifi_nrf_set_twt(const struct device *dev,
 		if (twt_params->teardown.teardown_all) {
 			if (vif_ctx_zep->neg_twt_flow_id == 0xFF) {
 				LOG_ERR("Invalid negotiated TWT flow id\n");
+				twt_params->fail_reason =
```
Comment (krish2718) [inline drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_mgmt.c:587]: This seems unrelated to the commit log, please remove or make it a separate commit.

Comment (ajayparida) [reply drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_mgmt.c:587]: Added a separate commit.

**Review by krish2718 — APPROVED** [review]

**Review by sachinthegreen — APPROVED** [review]

**Comment by krish2718** [comment]
merged to #11079 to make Zephyr manifest udpate easier.

**Comment by krish2718** [comment]
Consoildated this in to #11079 as this pulls latest `sdk-hostap` which needs latest `sdk-zephyr`
